packages/lib/bookingSuccessRedirect.ts

```diff
@@ -36,9 +36,36 @@ export const getBookingRedirectExtraParams = (booking: SuccessRedirectBookingTyp
     "location",
   ];
 
-  return (Object.keys(booking) as BookingResponseKey[])
+  const basicParams = (Object.keys(booking) as BookingResponseKey[])
     .filter((key) => redirectQueryParamKeys.includes(key))
     .reduce((obj, key) => ({ ...obj, [key]: booking[key] }), {});
+
+  const serializeResponseValue = (value: any): string => {
+    if (Array.isArray(value)) {
+      return value.join(",");
+    } else if (typeof value === "object" && value !== null) {
+      return encodeURIComponent(JSON.stringify(value));
+    } else {
+      return String(value);
+    }
+  };
+
+  const responseParams = booking.responses
+    ? Object.keys(booking.responses).reduce((obj, key) => {
+        obj[`responses.${key}`] = serializeResponseValue(booking.responses[key]);
+        return obj;
+      }, {} as Record<string, any>)
+    : {};
+
+  const meetingLink = booking.references?.find(
+    (ref) => ref.meetingUrl
+  )?.meetingUrl;
+
+  return {
+    ...basicParams,
+    ...responseParams,
+    ...(meetingLink ? { "responses.meetingLink": meetingLink } : {}),
+  };
```

Comment:
Good find but i think instead adding an extra param, overwrite the location when there is meetingLink present. Means location response value would be the meetingurl

Reply:
Updatedd

Reply:
Pls fix the type error now

Reply:
Which typo error check the latest commit please

Reply:
Have you checked the test to see if it has passed? Please check below

Reply:
Done